buildimage.py

```diff
@@ -472,11 +506,18 @@ def run_cmd(cmd):
     argparser.add_argument("--platform", choices=platformlist, default="raspbian", help="Select platforn type for image (default = \"raspbian\")")
     argparser.add_argument("--profile", choices=profilelist, default="ekstep", help="Select profile (default = \"ekstep\")")
```

Comment:
This would require some time and might introduce bugs which would again require another iteration of fixing-testing. I will take this as a follow-up in 1.16.